buildSrc/src/main/java/org/elasticsearch/gradle/precommit/TestingConventionsTasks.java

```diff
@@ -80,7 +76,9 @@ public TestingConventionsTasks() {
         return getProject().getTasks()
             .withType(Test.class)
             .stream()
-            .filter(t -> tasks == null || tasks.contains(t.getName()))
+            // hardcoded to the old test task names, this will change as they are converted to individual source sets
```

Comment:
Ug, gross. What are we filtering _out_ here? Just internal cluster tests?

Reply:
Anything that is from a sourceSet *other* than test. The testingConventions task is really about the overlay of multiple test tasks on the test sourceset. In the previous PR I handled this by allowing to override the test tasks that should be inspected by it, but that turns out to be cumbersome when adding in x-pack projects. I'm open to other ideas...

Reply:
We could, when grabbing the candidate class files for a task, filter that collection for only those class files that belong to the test source set.

Reply:
Worked perfectly, thanks!